recipe/meta.yaml

```
@@ -44,13 +55,17 @@ requirements:
   build:
     - cmake >=3.21.1,<4
     - {{ compiler('cxx') }}
+{% if cuda_present != "None" %}
     - {{ compiler('cuda') }}  # [cuda_compiler_version != "None"]
+{% endif %}
```

Is this still required with the introduction of `cuda_present`?
```suggestion
{% if cuda_present != "None" %}
    - {{ compiler('cuda') }}
{% endif %}
```

---

The jinja conditional `if cuda_present != "None"` is redundant because the selector `cuda_compiler_version != "None"` is already used. Please remove the `cuda_present` jinja variable. In most of the places where it was added, it is redundant.

---

If builds are failing locally for you because "cuda_compiler_version" is undefined, it is because you aren't including one of the build variant config files from the `.ci_support/` folder. These are the files that contain the pinnings for the packages on our channel including the compiler versions.

---

thanks for the hints, solved.